vulkan/vulkan_proc_table.h

```diff
@@ -67,6 +66,10 @@ class VulkanProcTable {
   Proc<PFN_vkDestroyFence> destroyFence;
   Proc<PFN_vkWaitForFences> waitForFences;
 
+#if OS_ANDROID
```

Comment:
To access any of the function prototypes, we ned to import `vulkan_interface.h`. That has to include `build_config.h`.